### DISCUSSION TITLE
How do you make a glass/translucent/transparent window with blur?

### DISCUSSION THREAD

**Discussion by TheKoTech** [abenz1267/walker#112] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/216f9667-1d6f-4312-bd3e-c265669be922)

This above is all i managed to do lol.

**Comment by abenz1267** [comment]
If using hyprland, put this in your hyprland.conf:

layerrule=blur,walker

Give Walker a transparent background without the blur.

**Reply by TheKoTech** [reply, marked as answer]
That works, but also blurs everything around. For anyone looking to make it blur only the window itself:

```
layerrule = ignorezero, walker
layerrule = blur,walker
```

![image](https://github.com/user-attachments/assets/1ee530aa-225b-46c0-b6eb-8719fb832331)

https://wiki.hyprland.org/Configuring/Window-Rules/#layer-rules

**Comment by ad-on-is** [comment]
```
layerrule=ignorezero, walker 
layerrule=blur,walker
layerrule=ignorealpha 0.5, walker
```
this worked for me
